ioctl/cmd/ins/ins.go

```diff
@@ -0,0 +1,67 @@
+package ins
+
+import (
+	"encoding/hex"
+
+	"github.com/spf13/cobra"
+	"golang.org/x/crypto/sha3"
+
+	"github.com/iotexproject/iotex-core/ioctl/config"
+)
+
+// IO_NODE hash for io
+const IO_NODE = "b2b692c69df4aa3b0a24634d20a3ba1b44c3299d09d6c4377577e20b09e68395"
+
+// Multi-language support
+var (
+	_insCmdShorts = map[config.Language]string{
+		config.English: "Manage INS of IoTeX blockchain",
+		config.Chinese: "管理IoTeX区块链上INS",
+	}
+	_flagInsEndPointUsages = map[config.Language]string{
+		config.English: "set endpoint for once",
+		config.Chinese: "一次设置端点", // this translation
+	}
+	_flagInsInsecureUsages = map[config.Language]string{
+		config.English: "insecure connection for once",
+		config.Chinese: "一次不安全连接", // this translation
+	}
+)
+
+// InsCmd represents the ins command
+var InsCmd = &cobra.Command{
+	Use:   "ins",
+	Short: config.TranslateInLang(_insCmdShorts, config.UILanguage),
+}
+
+func init() {
+	InsCmd.AddCommand(_insRegisterCmd)
+	InsCmd.PersistentFlags().StringVar(&config.ReadConfig.Endpoint, "endpoint",
+		config.ReadConfig.Endpoint, config.TranslateInLang(_flagInsEndPointUsages,
+			config.UILanguage))
+	InsCmd.PersistentFlags().BoolVar(&config.Insecure, "insecure", config.Insecure,
+		config.TranslateInLang(_flagInsInsecureUsages, config.UILanguage))
+}
+
+func nameHash(name string) (hash [32]byte, err error) {
+	var ioNode []byte
+	ioNode, err = hex.DecodeString(IO_NODE)
+	if err != nil {
+		return
+	}
+
+	sha := sha3.NewLegacyKeccak256()
+	if _, err = sha.Write(ioNode); err != nil {
+		return
+	}
+	nameSha := sha3.NewLegacyKeccak256()
+	if _, err = nameSha.Write([]byte(name)); err != nil {
+		return
+	}
```

Comment:
can we directly write name into existing `sha`?
```
sha.Write([]byte(name))
```

Reply:
Can't use the previous `sha`, because `namehash` need `nameSha` result as its input.

Reply:
oh i see, move L57-60 to in front of L53? think that's better readability